source/manual/document-collection-emails-a-special-case.html.md

```diff
@@ -0,0 +1,158 @@
+[email-signup-documentation]:/manual/email-signup-journeys.html.md
+[email-signup-skip-account-heading]:/manual/email-signup-journeys#how-does-the-skip-account-feature-work
+
+### What is a Taxonomy topic email override
+
+* A taxonomy_topic_email_overide is a type of link that [can only be added to document collections][document-collection-schema] **that have never previously been published**.
+* The field is set via a rake task in Whitehall.
+
+* The taxonomy_topic_email_overide stores the content_id of a taxonomy topic that a user will be subcribed to when signing up for emails on the document collection page.
+
+### How does the Taxonomy topic email override field work
+
+When rendering a document collection page, government frontend checks for the presence of a taxonomy_topic_email_override link in the content item.
+
+* If a link is found, the page is rendered with a [sign up link component][email-signup-documentation signup-link-heading] that will find or create a taxonomy topic email subscription for the topic stored in the field.
+
+* If no link is found, the page is rendered with a [single page notification button][email-signup-single-button-heading] that will sign the user up to a standard document collection email list instead.
+
+[document-collection-schema]: https://github.com/alphagov/publishing-api/blob/main/content_schemas/formats/document_collection.jsonnet#L69-L71
+
+[email-signup-documentation signup-link-heading]:/manual/email-signup-journeys#signup-link-component
+[email-signup-single-button-heading]:/manual/email-signup-journeys#single-page-notification-button-component
+[email-signup-subscription-link-heading]:/manual/email-signup-journeys#subscription-links-component
+[email-signup-subscription-link-heading]:/manual/email-signup-journeys#subscription-links-component
+[email_signup_component_guide]: https://components.publishing.service.gov.uk/component-guide/signup_link
+
+### How to set a taxonomy_topic_email_override
+
+**As mentioned above, it is only possible to set this field on document collections that have never been published before.**
+
+> Once a document is published with a taxonomy_topic_email_override, the field cannot be changed.
+
+#### Warning: Do not set this field unless you have an exceptionally good reason
+
+The following developer steps should **only** be taken if the request has comes from a content designer that has followed the guidance set out in [HMRC topic document collections: guidance for JCDs on 2nd line][content_documentation]. This is because once published, the field cannot be changed, so it's important that any government publisher requesting this feature has been talked through the risks and benefits before the override is set.
+
+[content_documentation]: https://docs.google.com/document/d/1MR5OaFG_DOCmWGL9o9MSGIPLMFe2mmSrV6Va-99cSzw/edit#heading=h.y6ckz26uo2lf
+
+#### 1. Find the content-id of the document collection
```

Comment:
Minor: I think this should say something other than "content-id" as that has a specific meaning. Perhaps document-collection-id or whitehall document-id?